nextflow_schema.json

```diff
@@ -71,34 +70,34 @@
                     "description": "Tsv indicating list of factors (conditions in the metadata table/samplesheet) and the new level on which to relevel the factor. Check contrasts docs."
                 },
                 "batch_effect": {
-                    "type": "string",
-                    "default": "false",
+                    "type": "boolean",
                     "description": "Turn on this flag if you wish to consider batch effects. You need to add the batch effect to the linear model too!"
                 },
                 "logFC_threshold": {
                     "type": "integer",
                     "default": 0,
                     "description": "Threshold (int) to apply to Log 2 Fold Change to consider a gene as differentially expressed."
                 },
-                "pval_threshold": {
+                "adj_pval_threshold": {
                     "type": "number",
                     "default": 0.05,
                     "description": "p value (float) to consider a gene as differentially expressed. The default value is 0.05."
                 },
-                "use_vst": {
-                    "type": "boolean",
-                    "default": false,
-                    "description": "Set this flag to true to use vst transformation (recommended when >50 samples; if false, will use rlog)"
+                "norm_method": {
+                    "type": "string",
+                    "default": "rlog",
+                    "enum": ["rlog", "vst", "vst-force"],
+                    "description": "Which transformation to use during DE analysis; either rlog or vst or vst-force . If vst: the pipeline will override this and run rlog if it detects a strong variation in size factors in the input data (this will be stated in the report). To disable this behavior, set to vst-force."
```

Comment:
```suggestion
                    "description": "Which transformation to use during DE analysis; either rlog or vst or vst-force. If vst: the pipeline will override this and run rlog if it detects a strong variation in size factors in the input data (this will be stated in the report). To disable this behavior, set to vst-force."
```